lib/wwwtxt/plain_text_renderer.rb

```diff
@@ -0,0 +1,123 @@
+require "htmlentities"
+
+module WwwTxt
+  class PlainTextRenderer
+    FORMAT = "txt"
+
+    def self.render_from_html(html)
+      strip_html(html)
+    end
```

Comment:
Instead of this, which just calls strip_html anyway, might I suggest...

```ruby
def self.strip_html(*args)
  new(*args).strip_html
end

private

def strip_html(html, from_charset = 'UTF-8')
  ...
end
```
`